src/status_im/ui/screens/wallet/events.cljs

```diff
@@ -224,3 +230,12 @@
   :wallet/show-error
   (fn []
     {:show-error (i18n/label :t/wallet-error)}))
+
+(handlers/register-handler-fx
+  :wallet/set-up-passed
+  (fn [{{:accounts/keys [accounts current-account-id] :as db} :db}]
+    (let [current-account (get accounts current-account-id)
+          new-account (assoc current-account :wallet-set-up-passed? true)]
+      {:db            (assoc-in db [:accounts/accounts current-account-id] new-account)
```

Comment:
you can use `status-im.ui.screens.accounts.events/account-update` it prepares fx